frontend/model/state.js

```diff
@@ -72,22 +70,23 @@ sbp('sbp/selectors/register', {
     //   state.notifications = []
     // }
 
+    const chatroomState = state.chatroom || {}
     // TODO: need to remove the whole content after we release 0.2.*
-    for (const chatRoomId in state.chatRoomUnread) {
-      if (!state.chatRoomUnread[chatRoomId].messages) {
-        state.chatRoomUnread[chatRoomId].messages = []
+    for (const chatRoomId in chatroomState.chatRoomUnread) {
+      if (!chatroomState.chatRoomUnread[chatRoomId].messages) {
+        chatroomState.chatRoomUnread[chatRoomId].messages = []
       }
-      if (state.chatRoomUnread[chatRoomId].mentions) {
-        state.chatRoomUnread[chatRoomId].mentions.forEach(m => {
-          state.chatRoomUnread[chatRoomId].messages.push(Object.assign({ type: MESSAGE_TYPES.TEXT }, m))
+      if (chatroomState.chatRoomUnread[chatRoomId].mentions) {
+        chatroomState.chatRoomUnread[chatRoomId].mentions.forEach(m => {
+          chatroomState.chatRoomUnread[chatRoomId].messages.push(Object.assign({ type: MESSAGE_TYPES.TEXT }, m))
         })
-        Vue.delete(state.chatRoomUnread[chatRoomId], 'mentions')
+        Vue.delete(chatroomState.chatRoomUnread[chatRoomId], 'mentions')
       }
-      if (state.chatRoomUnread[chatRoomId].others) {
-        state.chatRoomUnread[chatRoomId].others.forEach(o => {
-          state.chatRoomUnread[chatRoomId].messages.push(Object.assign({ type: MESSAGE_TYPES.INTERACTIVE }, o))
+      if (chatroomState.chatRoomUnread[chatRoomId].others) {
+        chatroomState.chatRoomUnread[chatRoomId].others.forEach(o => {
+          chatroomState.chatRoomUnread[chatRoomId].messages.push(Object.assign({ type: MESSAGE_TYPES.INTERACTIVE }, o))
         })
-        Vue.delete(state.chatRoomUnread[chatRoomId], 'others')
+        Vue.delete(chatroomState.chatRoomUnread[chatRoomId], 'others')
```

Comment:
I think we can delete this whole section now since the groups will need to be recreated anyway.

Reply:
Yeah, correct. As we can see in the comment above, we could clear the `state/vuex/postUpgradeVerification` as we have already published version 0.2.0.